tests/system/python/e2e/test_e2e_pi_scaleset.py

```diff
@@ -117,6 +129,28 @@ def test_end_to_end(self, start_south_north_with_filter, read_data_from_pi, fogl
         if not skip_verify_north_interface:
             self._verify_egress(read_data_from_pi, pi_host, pi_admin, pi_passwd, pi_db, wait_time, retries)
 
+        tracking_details = self.get_asset_tracking_details(foglamp_url, "Ingest")
+        assert len(tracking_details["track"]), "Failed to track Ingest event"
+        tracked_item = tracking_details["track"][0]
+        assert "Room #1" == tracked_item["service"]
+        assert "http-e1" == tracked_item["asset"]
+        assert "http_south" == tracked_item["plugin"]
+
+        tracking_details = self.get_asset_tracking_details(foglamp_url, "Filter")
+        assert len(tracking_details["track"]), "Failed to track Filter event"
+        tracked_item = tracking_details["track"][0]
+        assert "North v2 PI" == tracked_item["service"]
```

Comment:
should not this be filter? or its the service (task) to which `SS #1` filter is applied

Reply:
Its the task on which SS #1 filter is applied.